src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,6 +27,10 @@
 public class RobotContainer {
   private final DriveTrain m_driveTrain = new DriveTrain();
 
+
+  public static final XboxController xboxController = new XboxController(0);
+
+  public static final  PhotonVision camera = new PhotonVision();
```

Comment:
## Format

Okay, first and foremost, follow naming conventions, fields should start with an `m_`.

## Static

No field of `RobotContainer` should be static. Setting variables that every object can access is a dangerous game, you open the door to using it from places that should not have the ability to use it, and most terrifying, it can be used wrong without it being known. The next person to use the certain static variable might say oh thats a nice name let me _**DEFINE**_ this variable and write:

```java
RobotContainer.variable = new Variable()
```

Even though variable is already initialized by a different element of the robot code. Initializing again using the same variable will only delete the former value and replace it with the new one.

```suggestion
  public static final XboxController m_xboxController = new XboxController(0);
  public final PhotonVision m_camera = new PhotonVision();
```